publication.py

```diff
@@ -293,12 +289,12 @@ def get_last_modified_datetime(ctx, vault_package):
         "COLL_NAME = '" + vault_package + "' AND META_COLL_ATTR_NAME = '" + constants.UUORGMETADATAPREFIX + 'action_log' + "'",
         genquery.AS_LIST, ctx
     )
-
     for row in iter:
         log_item_list = jsonutil.parse(row[1])
-        my_date = datetime.fromtimestamp(int(log_item_list[0]))
 
-        # ISO8601-fy
+        import datetime
```

Comment:
With `from datetime import datetime` as top level import we can remove this import and change `datetime.datetime.fromtimestamp` below into `datetime.fromtimestamp`.